manyfews/calculations/tests.py

```diff
@@ -146,12 +146,12 @@ def test_calculation_GenerateRiverFlow(self):
         testDate = datetime.astimezone(testDate, tz=timezone(timedelta(hours=0)))
         nextDay = testDate + timedelta(days=1)
         runningGenerateRiverFlows(
-            dt=0.25, beginDate=testDate, dataLocation=testLocation
+            dt=0.25, predictionDate=testDate, dataLocation=testLocation
         )
 
         # extract result from data base.
-        RainAndEvapotranspirationData = RainAndEvapotranspiration.objects.all()
-        PotentialRiverFlowsData = PotentialRiverFlows.objects.all()
+        RiverFlowCalculationOutputData = RiverFlowCalculationOutput.objects.all()
+        RiverFlowPredictionData = RiverFlowPrediction.objects.all()
```

Comment:
And here - could these variables have names starting with `river`?

Reply:
accepted and modified